### PR TITLE
Changed the start-app test helper to use `Ember.assign`.

### PR DESCRIPTION
**Pull request by workmanw**
Some discussion here: ember-cli/ember-new-output#5 . I mistakenly submitted the PR to that repo not knowing it was generated.

### CODE REVIEW

**Comment by Turbo87** [comment]
@workmanw awesome, thanks!

File: blueprints/app/files/tests/helpers/start-app.js [hunk 1]
```diff
@@ -5,8 +5,7 @@ import config from '../../config/environment';
 export default function startApp(attrs) {
   let application;
 
-  let attributes = Ember.merge({}, config.APP);
-  attributes = Ember.merge(attributes, attrs); // use defaults, but you can override;
+  let attributes = Ember.assign({}, config.APP, attrs); // use defaults, but you can override;
```
Comment (nathanhammond) [inline blueprints/app/files/tests/helpers/start-app.js]: Let's move the comment to the previous line for line length reasons.

**Comment by nathanhammond** [comment]
LGTM with one modification.

**Comment by workmanw** [comment]
@nathanhammond 👍 Updated.

**Comment by nathanhammond** [comment]
@homu r+

**Pull request merged**